### DISCUSSION TITLE
Premium Packages Or Paid Packages

### DISCUSSION THREAD

**Discussion by kemzops** [npm/feedback#301] [Website] [answered]
i think npm should add some way to earn money from the packages for example spigotmc (have free plugins & paid plugins in the same time)

**Comment by Bunneyy** [comment]
I like the idea

**Reply by zz3jane** [reply]
me t00

**Comment by MylesBorins** [comment, marked as answer]
Hey @azizmjaber02 thanks for the suggestion.

Currently we have support with the [`npm fund`](https://docs.npmjs.com/cli/v7/commands/npm-fund) command to help let folks know ways they can help financially support the development of packages. We also have explicit policies around [commercial content](https://www.npmjs.com/policies/open-source-terms#commercial-content). Currently it is possible to distribute commercial code via npm, as long as you are not violating the terms of the commercial-content policy mentioned above.

What we don't have right now, and I think what you are asking for, is an on the rails marketplace experience to make it easy to purchase / license packages that are distributed on npm.

At this time I do not see us as npm prioritizing the development of a commercial marketplace for packages. The work we are prioritizing at the moment is related to improving the experience for Open Source Development, Package Publication, Consuming Open Source Packages, and general Operational Excellence.
